### ISSUE TITLE
thanks you

### ISSUE THREAD

**Issue by exotikcheetos** [0mWindyBug/KDP-compatible-driver-loader#2]
Hi, I would just like to know if this makes it possible to put the ioctl UD driver with this mapper ?

**Comment by 0mWindyBug** [comment]
Hey, I’m not sure what you mean by “put the ioctl UD driver” , could you rephrase that please ?

**Comment by exotikcheetos** [comment]
> Hey, I’m not sure what you mean by “put the ioctl UD driver” , could you rephrase that please ?

the mapper ensures that my ioctl driver will not be detected by the anticheat or really need EV cert ?

**Comment by 0mWindyBug** [comment]
All it does is get your unsigned driver loaded with DSE and KDP turned on .  it doesn’t remove any traces of the driver being loaded . It also doesn’t map anything tho, read the how it works section : it will “disable” DSE in a KDP compatible manner and load your driver via NtLoadDriver. From the it’s up to you to extend the functionality to remove traces (code for this already been shared  in UC) or make your driver appear legitimate.

**Comment by exotikcheetos** [comment]
> All it does is get your unsigned driver loaded with DSE and KDP turned on . it doesn’t remove any traces of the driver being loaded . It also doesn’t map anything tho, read the how it works section : it will “disable” DSE in a KDP compatible manner and load your driver via NtLoadDriver. From the it’s up to you to extend the functionality to remove traces (code for this already been shared in UC) or make your driver appear legitimate.

and just like that if I pay you he would be able to make me one i use IOCTL driver but i don't really want to buy a EV cert  because it can be easily blacklisted thanks alot

**Comment by 0mWindyBug** [comment]
contact me through discord ( 0xwindybug) as this is not an issue with the loader , thanks.
